CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ x.y.z Release notes (yyyy-MM-dd)
 * Xcode: 12.4-13.2.1.
 
 ### Internal
-* Upgraded realm-core from ? to ?
+* Upgraded realm-core from 11.8.0 to 11.9.0
```

Comment:
Also need changelog entries for the changes which come from bumping the core version.